src/applications/facility-locator/config.js

```diff
@@ -14,6 +15,7 @@ export const api = {
     credentials: 'include',
     headers: {
       'X-Key-Inflection': 'camel',
+      'Source-App-Name': manifest.entryName,
```

Comment:
This and GIBCT (below) use these config constants, which are declared before `startApp` is called, so `window.appName` would've been `undefined`. For these cases, we have to directly pull the `manifest.entryName`.

Reply:
Added comment in code just to be safe.